app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java

```diff
@@ -71,13 +72,16 @@
 
     private int sortType;
 
+    private TextView noSpeakersView;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noSpeakersView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630556/issues/source?bid=3233850&fileBranchId=3233897#l75)